README.md
The following tweaks are made.

## Beacons
vanilla beacon - 500kw power, 15 module slots, 50% transmission effectivity  
Advanced beacon - 2mw power, 30 module slots, 200% transmission effectivity  
Wide Area Beacon - 2mw power, 25 module slots, 75% transmission effectivity  
Wide Area Beacon 2 - 10mw power, 50 module slots, 100% transmission effectivity.  

Additionally, vanilla beacons are now allowed in space.

Additionally, if [Beacons but Better](https://mods.factorio.com/mod/walkable-beacons) is installed, its allow productivity modules in beacons setting is applied to the krastorio/space exploration beacons.

## Modules
While each tier requires 3 modules of the previous tier, each tier outputs 3 modules rather than 1.  Also,  each tier has a crafting time of 1.6^tier.  (tier 9 takes 68 seconds to craft.)

## Machines
Electrolysis plant has 5x crafting speed  
Filtration plant has 4x crafting speed  
Atmospheric condenser has 2x crafting speed.  
Crusher has 2x crafting speed.  
Mining drill mk3 has 2x mining speed, 6 module slots, and consumes 300kw of power.  
Core mining drill stacks to 5, and has 4x mining speed.  

## Tiles
Space platform scaffold recipe produces 10 units.  
Space platform plating recipe produces 10 units.  
Spaceship floor recipe produces 10 units.  
Recipes for the stone version of landfill, and the sand+water version of landfill produces 10 units each.  pure sand recipe intentionally left at 1 unit as a way to consume stone quickly.  

## Intermediate products
Heat shielding recipe produces 5 units.  
Low Density Structure recipe, with and without beryllium produces 5 units.  
All rocket-fuel recipes reverted to pre [space-exploration-postprocess](https://mods.factorio.com/mod/space-exploration-postprocess) 0.5.27.  

## Teleportation Techs
[Portals](https://mods.factorio.com/mod/Portals) now unlocks early  
[Teleporters](https://mods.factorio.com/mod/Teleporters) now unlocks early  

## Mining Drones and Transport Drones
Mining Depots can now be used in space to directly mine space resources such as Water Ice, Methane Ice and Naquim  
Transport drone roads can now be constructed directly on Empty space, without the intermediate step of first constructing scaffold or plating, since you would get those back anyways.  

## Others
Add Sand-Water landfill recipe when playing only space exploration  
Re-add the Hydrogen atmospheric condensation recipe when playing SE-K2.  


changelog.txt
---------------------------------------------------------------------------------------------------
Version: 1.0.5
Date: TBD
	- TBD
---------------------------------------------------------------------------------------------------
Version: 1.0.4
Date: 2022-02-17
	- Allow Space factory buildings on Spaceship
---------------------------------------------------------------------------------------------------
Version: 1.0.3
Date: 2022-01-30
	- Beacons can now be placed in space. No more need to use Beacons but Better + transport drone roads to work around the placement issue.
	- Mining Depots can now be placed in space.
	- Roads can now be directly constructed on empty space.
---------------------------------------------------------------------------------------------------
Version: 1.0.2
Date: 2022-01-29
	- Make Space Exploration optional
	- Make Krastorio 2 optional
	- Add settings.
	- Add Landfill with sand+water recipe when K2 is not installed.
	- Re-add hydrogen atmospheric condensation recipe.
---------------------------------------------------------------------------------------------------
Version: 1.0.1
Date: 2022-01-29
	- Allow early access to Teleporter and Portal Gun  
		(specifcally revert any changes space-exploration-postprocess made to these techs.)

---------------------------------------------------------------------------------------------------
Version: 1.0.0
Date: 2022-01-28
	- Initial Release


tweaks/beacons.lua
if settings.startup["cs2-tweaks-beacons"].value == true then
	data.raw.beacon.beacon.energy_usage = "500kW"
	data.raw.beacon.beacon.module_specification = {
	  module_info_icon_shift = { 0, 0.5 },
	  module_info_max_icons_per_row = 5,
	  module_info_max_icon_rows = 3,
	  module_info_multi_row_initial_height_modifier = -0.3,
	  module_slots = 15,
	  module_info_icon_scale = 0.35
	}
	

	if mods["Krastorio2"] then
		data.raw.beacon["kr-singularity-beacon"].energy_usage = "2000kW"
		data.raw.beacon["kr-singularity-beacon"].module_specification = {
		  module_info_icon_shift = { 0, 0.5 },
		  module_info_max_icons_per_row = 6,
		  module_info_max_icon_rows = 5,
		  module_info_multi_row_initial_height_modifier = -0.5,
		  module_slots = 30,
		  module_info_icon_scale = 0.25
		}
		data.raw.beacon["kr-singularity-beacon"].distribution_effectivity = 2
	end

	if mods["space-exploration"] then
		--local data_util = require("__space-exploration-postprocess__/data_util")
		--data.raw.beacon["beacon"].
		collision_mask_util_extended.remove_layer(data.raw["beacon"]["beacon"].collision_mask, space_collision_layer)
		collision_mask_util_extended.remove_layer(data.raw["beacon"]["beacon"].collision_mask, spaceship_collision_layer)
		
		data.raw.beacon["se-wide-beacon"].energy_usage = "2000kW"
		data.raw.beacon["se-wide-beacon"].module_specification = {
		  module_info_icon_shift = { 0, 0.5 },
		  module_info_max_icons_per_row = 7,
		  module_info_max_icon_rows = 4,
		  module_info_multi_row_initial_height_modifier = -0.3,
		  module_slots = 25,
		  module_info_icon_scale = 0.35
		}
		data.raw.beacon["se-wide-beacon"].distribution_effectivity = 0.75

		data.raw.beacon["se-wide-beacon-2"].energy_usage = "10000kW"
		data.raw.beacon["se-wide-beacon-2"].module_specification = {
		  module_info_icon_shift = { 0, 0 },
		  module_info_max_icons_per_row = 10,
		  module_info_max_icon_rows = 5,
		  module_info_multi_row_initial_height_modifier = -0.3,
		  module_slots = 50,
		  module_info_icon_scale = 0.25
		}
		data.raw.beacon["se-wide-beacon-2"].distribution_effectivity = 1
	end
end

if mods["walkable-beacons"] then
	if settings.startup["walkable-beacons-allow-productivity"].value == true then
		if mods["Krastorio2"] then
			data.raw["beacon"]["kr-singularity-beacon"].allowed_effects = {"consumption", "speed", "pollution", "productivity"}
		end
		if mods["space-exploration"] then
			data.raw["beacon"]["se-wide-beacon"].allowed_effects = {"consumption", "speed", "pollution", "productivity"}
			data.raw["beacon"]["se-wide-beacon-2"].allowed_effects = {"consumption", "speed", "pollution", "productivity"}
		end
	end
end